### ISSUE TITLE
Shader compilation fails and everything is rendered as black on Mac

### ISSUE THREAD

**Issue by yvt** [Hypersomniac/blender-shader#7]
I'm using v0.4 and everything is rendered as black, no matter what shader nodes are used.

[System information](https://gist.github.com/yvt/f0003fa0367b465404c4e45c4cac2804)

<img width="1795" alt="screen shot 2016-06-08 at 1 21 23 am" src="https://cloud.githubusercontent.com/assets/5253988/15865901/83c66fce-2d17-11e6-984e-508fc0b96e78.png">

```
Read new prefs: /Users/tcpp/Library/Application Support/Blender/2.77/config/userpref.blend
found bundled python: /Users/tcpp/Downloads/blender_pbr/blender.app/Contents/MacOS/../Resources/2.77/python
read blend: /Users/tcpp/3D/JS/JS.blend
GPUShader: compile error:
ERROR: 0:2592: Invalid call of undeclared identifier 'texelFetch'
ERROR: 0:2595: Use of undeclared identifier 'depth'
ERROR: 0:2598: Use of undeclared identifier 'depth'
ERROR: 0:2603: Invalid call of undeclared identifier 'texelFetch'
ERROR: 0:2606: Use of undeclared identifier 'depth'
ERROR: 0:2609: Use of undeclared identifier 'depth'
ERROR: 0:3232: Invalid call of undeclared identifier 'texelFetch'
ERROR: 0:3243: Invalid call of undeclared identifier 'texelFetch'
ERROR: 0:7211: Invalid call of undeclared identifier 'texelFetch'
```

**Comment by Hypersomniac** [comment]
Thank you for reporting. I know the problem. It will be fixed in a week (the time to get back to my computer).

**Comment by Hypersomniac** [comment]
Hopefully fixed in 4cae0cb418660aef4b47b19a0d89168759ddf9b3 and ddfae3d2e1bdfee87531691c99c15b230c790e82
